src/store/index.ts

```diff
@@ -1,15 +1,20 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import { arrowStore } from '@/store/modules/arrow/arrow'
+import { scoreStore } from '@/store/modules/score/score'
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
-  state: {
-  },
   mutations: {
```

Comment:
I suppose removing empty mutations, actions and getters won't break anything. I can be wrong...

Reply:
I`ll check it and if it unnecessary i`ll fix it in next commit